src/groovy/au/org/emii/gogoduck/json/JSONSerializer.groovy

```diff
@@ -79,6 +79,7 @@ class JSONSerializer {
             'constraints',
             'grailsApplication',
             'aggrUrl',
+            'statusUrl',
```

Comment:
I don't think it's required to save the job URL in the JSON. Better to build it on the fly when it's required - i.e. when sending the email.

After all the only thing you need to build the URL is just the UUID of the job.